<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="Image_Window">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>550</y>
      <width>380</width>
      <height>200</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>图像窗口操作面板</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="Text_Camera_Calibration">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>115</y>
       <width>360</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>相机标定结果：1像素 =               毫米</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Extract_Region">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>65</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>提取区域</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Load_Image">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>载入图像</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Injection_Port_Detection">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>65</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注液口检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Binarization">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>65</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>二值化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Camera_Calibration">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>相机标定</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Camera_Calibration_Result">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>121</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Text_Workpiece_Detection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>360</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>物料检测结果：      有               无</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Exist">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>155</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Lack">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>155</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Injection_Port_Window">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>760</y>
      <width>380</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>注液口操作面板</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="Injection_MBR">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>最小外接矩形</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Injection_Image_Full">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>图像填充</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Injection_Contour_Extraction">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>轮廓提取</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Injection_Center_Point_Result">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>62</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Injection_Perpendicular_Bisector_Result">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>107</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Injection_Port_Center_Point">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>65</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>注液口中心点：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Injection_Perpendicular_Bisector">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>120</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>中垂线方程：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Result_display">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>700</y>
      <width>380</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>结果显示面板</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="Catch_Point">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>抓取点</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rotation_Angle">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>65</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>旋转角度</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Catch_Point_Result">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Rotation_Angle_Result">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>62</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="VC_Number">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>85</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>VC板计数：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="VC_Number_Result">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>110</y>
       <width>280</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="digitCount">
      <number>25</number>
     </property>
     <property name="mode">
      <enum>QLCDNumber::Dec</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>630</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>整体图像</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QLabel" name="Overall_Image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>612</width>
       <height>512</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>0</y>
      <width>630</width>
      <height>540</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>检测图像</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <widget class="QLabel" name="Detection_Image">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>612</width>
       <height>512</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="Program_Running_Status">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>20</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;程&lt;/p&gt;&lt;p&gt;序&lt;/p&gt;&lt;p&gt;运&lt;/p&gt;&lt;p&gt;行&lt;/p&gt;&lt;p&gt;状&lt;/p&gt;&lt;p&gt;态&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="Result_Program_Running_Status">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>560</y>
      <width>470</width>
      <height>340</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="Cover_Window">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>550</y>
      <width>380</width>
      <height>150</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>盖板操作面板</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="Cover_MBR">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>最小外接矩形</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cover_Image_Full">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>图像填充</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Cover_Contour_Extraction">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>轮廓提取</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Cover_Center_Point_Result">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>62</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="Cover_Line_Equation_Result">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>107</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="Cover_Center_Point">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>65</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>盖板中心点：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Cover_Line_Equation">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>110</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>轮廓线方程：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Result_display_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>850</y>
      <width>380</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="title">
     <string>进入子界面</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QPushButton" name="Catch_Point_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>相机界面</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Rotation_Angle_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>楷体</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>系统硬件界面</string>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_5</zorder>
   <zorder>Image_Window</zorder>
   <zorder>Injection_Port_Window</zorder>
   <zorder>Result_display</zorder>
   <zorder>groupBox_6</zorder>
   <zorder>Program_Running_Status</zorder>
   <zorder>Result_Program_Running_Status</zorder>
   <zorder>Cover_Window</zorder>
   <zorder>Result_display_2</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
